src/main/webapp/style.css

```diff
@@ -0,0 +1,236 @@
+body {
+  background-color: #ececec;
+  color: black;
+  font-family: Aksara Bali Galang;
+}
+
+main {
+  margin: auto;
+  max-width: 1035px;
+  width: 100%;  
+}
+
+#content {
+  margin: auto;
+  max-width: 100%;
+}
+
+h1 {
+  color: white;
+  font-family: Ruluko;
+  font-size: 40px;
+  font-weight: normal;
+  line-height: 69px;
+  margin: 10px 20px;
+}
+
+h2,
+h3 {
+  color: #856c8b;  
+  text-align: center;
+}
+
+h5 {
+  color: #787878;  
+  display: block;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: normal;
+  height: 26px;
```

Comment:
Can you remove this and address with line-height, margin, and/or padding instead?